integration/httpd_test.go

```diff
@@ -98,18 +104,7 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 					Execute(image.ID)
 				Expect(err).NotTo(HaveOccurred())
 
-				Eventually(func() string {
-					cLogs, err := docker.Container.Logs.Execute(container.ID)
-					Expect(err).NotTo(HaveOccurred())
-					return cLogs.String()
-				}).Should(
-					And(
-						MatchRegexp(`Apache\/\d+\.\d+\.\d+ \(Unix\) configured -- resuming normal operations`),
-						MatchRegexp(`fpm is running, pid \d+`),
-						ContainSubstring(`ready to handle connections`),
-					),
-				)
-				Eventually(container).Should(Serve(ContainSubstring("Hello World!")).OnPort(8080))
+				Eventually(container).Should(Serve(ContainSubstring("SUCCESS: date loads.")).OnPort(8080).WithEndpoint("/index.php?date"))
```

Comment:
Just to confirm, is this change predicated on the idea that if the endpoint is hit successfully fpm must also be working correctly?

Reply:
yeah, you got it. Without fpm running the php wouldn't render successfully.